query/src/lib.rs

```diff
@@ -35,6 +35,7 @@ extern crate mentat_core;
 
 use std::collections::{
     BTreeSet,
+    HashSet,
```

Comment:
Fold this down, especially 'cuz the earlier commit did the wrong thing with placeholders (I think).